frontend/packages/dev-console/src/utils/resource-label-utils.ts

```diff
@@ -72,6 +72,18 @@ export const getTriggerAnnotation = (
   ]),
 });
 
+export const getUserAnnotations = (allAnnotations: { [key: string]: string }) => {
+  const defaultAnnotations = [
+    'app.openshift.io/vcs-uri',
+    'app.openshift.io/vcs-ref',
+    'openshift.io/generated-by',
+    'image.openshift.io/triggers',
+    'alpha.image.policy.openshift.io/resolve-names',
+    'jarFileName',
+  ];
```

Comment:
So we dropped `'kubectl.kubernetes.io/last-applied-configuration'` as well in the [Pipeline world](https://github.com/openshift/console/blob/master/frontend/packages/pipelines-plugin/src/components/pipelines/modals/common/utils.ts#L84-L95).

Reply:
You meant that this annotation should be added here too?

Reply:
Hmm I think we might want to remove it as I saw this annotation in other resources as well though we dont add this specifically from our add flows but it might interfere with updating the resources like it probably did with `PipelineRuns` though I am not 100% sure.

Reply:
I think updates are different than creates. Updates I wanna say are good with this annotation being echoed because it's an update. But I don't think you can create a resource with it... I should try to test once I get a moment today to make sure my representation of the world is actually correct lol.

Reply:
We are not creating any resource with this annotation from add flows and I dont think that we have a similar case as `PipelineRuns` here or I am not aware of it.

Reply:
Hmmm, okay, let's ignore this thread -- I'm having issues reproducing it but my cluster went down as I was playing with various K8s / CRD kinds. Clearly this doesn't seem to fit the issue at hand, at least not directly.